aten/src/ATen/core/NamedTensor.cpp

```diff
@@ -1,6 +1,7 @@
+#define TORCH_ASSERT_NO_OPERATORS
```

Comment:
We should do this more often; I'm super annoyed NativeFunctions.h is accessible from ATen.h

Reply:
Oh interesting, that really shouldn't be public. At this point I guess it would break the world to remove though?

Reply:
It will be somewhat disruptive, as people often do `native::blah` to bypass the dispatcher, but I think it's worth fixing